package.json

```diff
@@ -11,7 +11,8 @@
     "postdoc": "grunt copy:jsduck",
     "selenium": "bash ./tests/selenium/selenium.sh",
     "selenium-daily": "MW_SCRIPT_PATH=/w npm run selenium-test -- --mochaOpts.grep @daily",
-    "selenium-test": "wdio ./tests/selenium/wdio.conf.js"
+    "selenium-test": "wdio ./tests/selenium/wdio.conf.js",
+    "jest-test": "jest -c ./tests/playwright/jest.e2e.config.js"
```

Comment:
Add scripts:

`playwright:open`: Run in non-headless mode
`playwright:close`: Run in headless mode